app/uk/gov/hmrc/helptosave/services/HelpToSaveService.scala

```diff
@@ -175,7 +175,11 @@ class HelpToSaveServiceImpl @Inject() (helpToSaveProxyConnector: HelpToSaveProxy
                 pagerDutyAlerting.alert("Could not parse JSON in eligibility check response")
                 Left(e)
             }, res => {
-              logger.debug(s"Call to check eligibility successful, received 200 (OK) ${timeString(time)}", nino, additionalParams)
+              if (res.resultCode === 4) {
+                metrics.itmpEligibilityCheckErrorCounter.inc()
```

Comment:
just wondering if we should log something/improve the logging here?

Reply:
I don't thing we need to, the response is already logged at INFO by the statement before the match at line 168